design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,210 @@
+}
+
+type OrderItemInterface {
+    eligible_for_return: Boolean @doc(description: "Indicates whether the order item is eligible for return.")
+}
+
+type Returns {
+    items: [Return] @doc(description: "List of returns")
+    page_info: SearchResultPageInfo @doc(description: "Pagination metadata")
+    total_count: Int @doc(description: "Total count of customer returns")
+}
+
+type Return @doc(description: "Customer return") {
+    id: ID! @doc(description: "Return number")
+    order_id: ID! @doc(description: "ID of the order based on which the return was created.")
+    creation_date: String! @doc(description: "The date when the return was requested.")
+    customer_email: String! @doc(description: "Email of the customer who created the return.")
+    customer_name: String @doc(description: "The name of the person returning the item")
+    status: ReturnStatus @doc(description: "Return status.")
+    shipping: ReturnShipping @doc(description: "Shipping information for the return.")
+    comments: [ReturnComment] @doc(description: "A list of comments posted for the return.")
+    items: [ReturnItem] @doc(description: "A list of items being returned.")
+    available_shipping_carriers: [ReturnShippingCarrier] @doc(description: "A list of shipping carriers available for returns.")
+}
+
+type ReturnItem {
+    id: ID!
+    product: ProductInterface! @doc(description: "The product being returned.")
+    custom_attributes: [CustomAttribute] @doc(description: "Return item custom attributes, which are marked by the admin to be visible on the storefront.")
+    request_quantity: Float! @doc(description: "The quantity of the item requested to be returned.")
+    quantity: Float! @doc(description: "The quantity of the items authorized by the merchant to be returned .")
+    status: ReturnItemStatus! @doc(description: "The return status of the item being returned.")
+}
+
+# See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/custom-attributes-container.md
+type CustomAttribute {
+    id: ID!
+    label: String!
+    value: String! @doc(description: "JSON encoded value of the attribute.")
+}
+
+type ReturnComment {
+    id: ID! @doc(description: "Comment ID.")
+    author_firstname: String! @doc(description: "First name of the user who posted the comment.")
+    author_lastname: String! @doc(description: "Last name of the user who posted the comment.")
+    created_at: String! @doc(description: "The date and time when the comment was posted.")
+    text: String! @doc(description: "The comment text.")
+}
+
+type ReturnShipping {
+    address: ReturnShippingAddress @doc(description: "Return shipping address, which is specified by the admin.")
+    tracking(id: ID): [ReturnShippingTracking] @doc(description: "Tracking information for all or a single tracking record when ID is provided.")
+}
+
+type ReturnShippingCarrier {
+    id: ID!
```

Comment:
What does `ID` mean? Usually, we represent shipping carriers like carrier + shipping method names.

Reply:
Same question, let's discuss here https://github.com/magento/architecture/pull/397#discussion_r460143090